CONTRIBUTING.md

```diff
@@ -165,3 +165,65 @@ general guidelines for the community as a whole:
   respond to an email you are potentially sending to a large number of
   people.  Please consider this before you update.  Also remember that
   nobody likes spam.
+
+## FAQ
+
+**Is it fine to go with Kotlin for my contribution?**
+
+We don't use Kotlin. This is due to the fact, that our main product is written in pure java.
+Even tough the Android community is adapting Kotlin well, most guys from the Java community
+have never seen Kotlin code before.
```

Comment:
I don't like the phrase "have never seen Kotlin code before". I guess most of them have seen Kotlin code, but aren't used to it.